oauth2-useragent-core/src/main/java/com/microsoft/alm/oauth2/useragent/AuthorizationResponse.java

```diff
@@ -61,6 +61,10 @@ public static AuthorizationResponse fromString(final String s) throws Authorizat
             for (final String pair : pairs) {
                 final String[] nameAndValue = NAME_VALUE_SEPARATOR.split(pair, 2);
                 try {
+                    if (nameAndValue.length != 2) {

```

Comment:
Nope, this fix isn't for what's returned from server.  This is actually to make the test work.  The unit test 'cut' returns something and when we try to parse it, we get an ArrayIndexOutOfBoundsException, this is to guard for that exception.  

I haven't seen a real life example that we would hit this yet.